src/agents/plots_videos.jl

```diff
@@ -73,7 +73,11 @@ evolving the ABM and a heatmap in parallel with only a few lines of code.
 """
 function abm_plot(model; resolution = (600, 600), kwargs...)
     fig = Figure(; resolution)
-    ax = fig[1,1][1,1] = Axis(fig)
+    ax = fig[1,1][1,1] = if length(size(model.space.s)) == 3
```

Comment:
Doesn't this explicitly assume the space is DiscreteSpace? I think it is much better if you code a function `dimensionality` that returns the space dimensionality.

Reply:
Yes! I missed that. Should I add this function in the pathfinding PR or a different one so it can be merged sooner?